.eslintrc.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  env: {
+    browser: true,
+    es6: true
+  },
+  extends: ["eslint:recommended", "plugin:prettier/recommended"],
+  parserOptions: {
+    sourceType: "module"
+  },
+  rules: {
+    indent: ["error", 2],
```

Comment:
These all seem like code formatting rules that should be already enforced by "plugin:prettier/recommended".
Could they just be removed?

Reply:
Will do a crosscheck on these